x/txfees/types/keys.go

```diff
@@ -11,16 +11,16 @@ const (
 	// RouterKey is the message route for slashing.
 	RouterKey = ModuleName
 
-	// FeeCollectorName the module account name for the fee collector account address.
-	FeeCollectorName = "fee_collector"
```

Comment:
This has tripped me up for 20+ mins on 2 separate occasions so I am finally fixing it. 

This is not a txfee module account, its just the name of the auth module account that we use to distribute to stakers with. We should never do this, and just import to module account name straight from auth, which I do in this PR.

Reply:
Ah I see, good job just using the authtypes instead.